Automattic/wp-calypso
# Hosting Command Palette: Show error notice when copy or reset fails

Fixes https://github.com/Automattic/dotcom-forge/issues/4825

## Proposed Changes

Displays an error message when 'Copy SSH connection string' or 'Reset SFTP/SSH password' fails:

![CleanShot 2023-12-14 at 06 10 05@2x](https://github.com/Automattic/wp-calypso/assets/36432/e0210814-bcca-493d-b336-8f2eb48917a0)

![CleanShot 2023-12-14 at 06 06 20@2x](https://github.com/Automattic/wp-calypso/assets/36432/cd8b491e-0cf2-4afb-a125-89468eeb6087)

It also:
* Removes the automatic redirect to Hosting Configuration because I think that's unexpected and inconsistent behavior. It's unexpected because the command performs an action. It's inconsistent because we don't automatically redirect for other action commands.
* Standardizes strings to use "SFTP/SSH".

## Testing Instructions

1. Create a new Business site and take it Atomic.
2. Open the command palette, choose 'Copy SSH connection string', select your new Business site, and verify the error message appears as expected.
3. Open the command palette, choose 'Reset SFTP/SSH password', select your new Business site, and verify the error message appears as expected.
4. Create a SFTP user for your site.
5. Verify the two aforementioned commands continue to work as expected.
